apis/database/v1alpha2/cloudsql_instance_types.go

```diff
@@ -151,66 +151,11 @@ type CloudsqlInstance struct {
 	Status CloudsqlInstanceStatus `json:"status,omitempty"`
 }
 
-// SetBindingPhase of this CloudsqlInstance.
-func (i *CloudsqlInstance) SetBindingPhase(p runtimev1alpha1.BindingPhase) {
-	i.Status.SetBindingPhase(p)
-}
-
-// GetBindingPhase of this CloudsqlInstance.
-func (i *CloudsqlInstance) GetBindingPhase() runtimev1alpha1.BindingPhase {
-	return i.Status.GetBindingPhase()
-}
-
-// SetConditions of this CloudsqlInstance.
-func (i *CloudsqlInstance) SetConditions(c ...runtimev1alpha1.Condition) {
-	i.Status.SetConditions(c...)
-}
-
-// SetClaimReference of this CloudsqlInstance.
-func (i *CloudsqlInstance) SetClaimReference(r *corev1.ObjectReference) {
-	i.Spec.ClaimReference = r
-}
-
-// GetClaimReference of this CloudsqlInstance.
-func (i *CloudsqlInstance) GetClaimReference() *corev1.ObjectReference {
-	return i.Spec.ClaimReference
-}
-
-// SetNonPortableClassReference of this CloudsqlInstance.
-func (i *CloudsqlInstance) SetNonPortableClassReference(r *corev1.ObjectReference) {
-	i.Spec.NonPortableClassReference = r
-}
-
-// GetNonPortableClassReference of this CloudsqlInstance.
-func (i *CloudsqlInstance) GetNonPortableClassReference() *corev1.ObjectReference {
-	return i.Spec.NonPortableClassReference
-}
-
 // GetProviderReference of this CloudsqlInstance
 func (i *CloudsqlInstance) GetProviderReference() *corev1.ObjectReference {
 	return i.Spec.ProviderReference
 }
```

Comment:
This looks suspiciously like part of the `resource.Managed` method set due to its naming and position in the file, but it's actually something only `CloudsqlInstance` implements.